kubernetes/samples/scripts/create-weblogic-domain-on-azure-kubernetes-service/azure-file-pv-template.yaml

```diff
@@ -24,3 +24,4 @@ spec:
   - uid=1000
   - gid=1000
   - mfsymlinks
+  - nobrl
```

Comment:
Where is this work-around documented?

So far, it looks like 'nobrl' disables locking - which has risks depending on the use case.

Reply:
It does disable locking - some folks tried this with sql-lite, which similarly uses byte range locks, and ran into corruption issues:   https://github.com/docker/for-win/issues/694

Reply:
Continuing discussion in slack.